src/github/handlers/repository-dispatch.ts

```diff
@@ -1,16 +1,13 @@
-import { GitHubContext } from "../github-context";
-import { dispatchWorker, dispatchWorkflow, getDefaultBranch } from "../utils/workflow-dispatch";
 import { Value } from "@sinclair/typebox/value";
-import { PluginInput, PluginChainState, expressionRegex, pluginOutputSchema } from "../types/plugin";
+import { GitHubContext } from "../github-context";
+import { expressionRegex, PluginChainState, PluginInput, pluginOutputSchema } from "../types/plugin";
 import { isGithubPlugin } from "../types/plugin-configuration";
-import { handleActionValidationWorkflowCompleted } from "./push-event";
+import { dispatchWorker, dispatchWorkflow, getDefaultBranch } from "../utils/workflow-dispatch";
 
 export async function repositoryDispatch(context: GitHubContext<"repository_dispatch">) {
   console.log("Repository dispatch event received", context.payload.client_payload);
 
-  if (context.payload.action === "configuration_validation") {
-    return handleActionValidationWorkflowCompleted(context);
-  } else if (context.payload.action !== "return_data_to_ubiquibot_kernel") {
+  if (context.payload.action !== "return_data_to_ubiquibot_kernel") {
```

Comment:
Let's deprecate `ubiquibot`

Reply:
This has be to changed in the kernel first then, I'll update it.